### PR TITLE
Andrius please review

### PR DESCRIPTION
**Pull request by d1j**

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log(1)
```
Comment (Andriusjok) [inline test.js]: Is this needed? Seems unrelated to the rest of the code

File: for_andrius/main.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import sys, hashlib
+from bitcoin.rpc import RawProxy
```
Comment (Andriusjok) [inline for_andrius/main.py:2]: And empty line after imports

File: for_andrius/main.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+p=RawProxy() # This reverses and then swaps every other char
```
Comment (Andriusjok) [inline for_andrius/main.py]: Please choose a better naming convention

File: for_andrius/main.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+def swap_order(param):
+    ba = bytearray.fromhex(param)
```
Comment (Andriusjok) [inline for_andrius/main.py]: Please rename to someting like byte_array

**Review by Andriusjok — COMMENTED** [review]
Prastas PR, iš tavęs tikėjausi daugiau.

File: test.js (hunk 1 above)
Comment (d1j) [reply test.js]: Sorry my man, removed.

File: for_andrius/main.py (hunk 2 above)
Comment (d1j) [reply for_andrius/main.py:2]: ?

File: for_andrius/main.py (hunk 3 above)
Comment (d1j) [reply for_andrius/main.py]: done

File: for_andrius/main.py (hunk 4 above)
Comment (d1j) [reply for_andrius/main.py]: done

**Review by Andriusjok — APPROVED** [review]
Super, susitvarkei su užduotimi!
